### DISCUSSION TITLE
Separate admin and user login form or no?

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#44163] [Q&A]
Do you think it is better to use a login form for the security of a project or two login forms, one for administrators and the other for users?

**Comment by karaka200** [comment]
> Do you think it is better to use a login form for the security of a project or two login forms, one for administrators and the other for users?

Some people say that in order to increase the security of the site, it is better for the admin login page to be separate from the user login page. It is also better to use two firewalls for this issue.
For example, in WordPress or Drupal, the admin login page is separate from the user login page.
What is your opinion?

**Comment by alessandro-podo** [comment]
have a look at this Discussion: https://github.com/symfony/symfony/discussions/43670

**Reply by karaka200** [reply]
Why did I think I did not ask this question? Now my practical project has started and that's why I wanted to be sure, but I forgot I had asked this before. Thank you for the reminder
